scripts/dev_cli.sh

```diff
@@ -351,6 +354,28 @@ cmd_build-container() {
 	   $BUILD_DIR
 }
 
+cmd_shell() {
+    say_warn "Starting a privileged shell prompt as root ..."
+    say_warn "WARNING: Your $CLH_ROOT_DIR folder will be bind-mounted in the container under $CTR_CLH_ROOT_DIR"
+    $DOCKER_RUNTIME run \
+	   -ti \
+	   --workdir "$CTR_CLH_ROOT_DIR" \
+	   --rm \
+	   --privileged \
+	   --security-opt seccomp=unconfined \
+	   --ipc=host \
+	   --net=host \
+	   --volume /dev:/dev \
```

Comment:
Is it on purpose that you didn't add `--mount type=tmpfs,destination=/tmp \`?

Reply:
Yes, either to do this or I can also see logic in bind mounting the host `/tmp` in. But just using the one in the container rootfs is probably not right.

Reply:
Done. Some of the unit tests need an executable `/tmp`, so I passed the `--tmpfs /tmp:exec` option.